prime-router/src/main/kotlin/cli/tests/BasicTests.kt

```diff
@@ -44,10 +44,11 @@ class Ping : CoolTest() {
             payloadName = "$name ${status.description}",
         )
         echo("Response to POST: $responseCode")
-        echo(json)
+        echo("Response message: $json")
         if (responseCode != HttpURLConnection.HTTP_OK) {
             bad("Ping/CheckConnections Test FAILED:  response code $responseCode")
-            exitProcess(-1) // other tests won't work.
+            outputAllMsgs()
+            exitProcess(1) // other tests won't work.
```

Comment:
What's this flip to `1`? Does this allow the ping test to fail?

Reply:
Unix style exit codes are 0 for success and normally a positive integer for errors.  It won't change a thing going from -1 to 1, but it is a bit more correct.

Reply:
This should probably not be a magic number. I think there are constants for return values.

Reply:
There are no constants defined for this either in Java, Kotlin or this baseline.  Other exit values in the baseline are set as hard coded numbers.

Reply:
Just another way in which Java disappoints me. How hard would it have been for them to include `std::EXIT_SUCCESS` from C++?

Reply:
I added an enum for this.